src/tweenable.js

```diff
@@ -1,5 +1,5 @@
 import * as easingFunctions from './easing-functions'
-import * as token from './token'
+import './token'
```

Comment:
🤔 Why is token being imported here in the first place? I cannot see any side effects inside of that module, so I don't understand how it could affect tweenable.js in any way

Reply:
That's a really good catch! It looks like this line was moved to here from `index.js` in the PR this is based on: https://github.com/jeremyckahn/shifty/pull/135/files#diff-8f8b83e7046411df901d4a840765f0d54656d50d127a18c5613a3e3aa3b2dd4b

In Shifty's `master` branch, this import ensures that `token` filters are attached to the `Tweenable` object:

https://github.com/jeremyckahn/shifty/blob/199774dc8e03b7de1a1971997193e692a5626fe1/src/index.js#L6-L8

It seems that the token import may have been moved to `Tweenable.js` inadvertently, so I've removed it in https://github.com/jeremyckahn/shifty/commit/62b0c6cc6bc646df806064c7037c8e4646c8c20a.